resources/views/admin/audio-controls.blade.php

```diff
@@ -0,0 +1 @@
+<audio controls src="{{ $src }}" @style(['width: 100%'])></audio>
```

Comment:
Add a catch here in case there is no `$src`. E.g., `$sound->content` is empty.

Reply:
If there's no `src` the `<audio>` tag degrades pretty gracefully:
![Screenshot 2023-08-25 at 12 51 44 PM](https://github.com/art-institute-of-chicago/aic-mobile-cms/assets/2098951/6e235c3c-cd79-4e70-981f-1e1172a21a4c)

Is there something else you'd like to see instead?

Reply:
It would be more clear to the user of the player didn't display at all of there was no audio to play. So maybe add an `@if` condition around the `<audio>` tag that only shows it if there's audio to play.